docs/src/main/asciidoc/transaction.adoc

```diff
@@ -367,37 +371,41 @@ which is a CDI bean and can be ``@Inject``ed.
 [[jdbcstore]]
 == Configure storing of Quarkus transaction logs in a database
 
-In cloud environments where persistent storage is not available, such as when application containers are unable to use persistent volumes, you can configure the transaction management to store transaction logs in a database by using a JDBC datasource.
+In cloud environments where persistent storage is not available, such as when application containers are unable to use persistent volumes, you can configure the transaction management to store transaction logs in a database by using a Java Database Connectivity (JDBC) datasource.
 
-IMPORTANT: While there are several benefits to using a database to store transaction logs, you might notice a reduction in performance compared with using the file system to store the logs.
+However, in cloud-native apps, it is important to assess transactions after careful evaluation.
+The `narayana-jta` extension, which manages these transactions, requires stable storage, a unique reusable node identifier, and a steady IP address to work correctly.
+While the JDBC object store provides a stable storage, users must still plan how to meet the other two requirements.
 
-Quarkus allows the following JDBC-specific configuration of the object store included in `quarkus.transaction-manager.object-store.<property>` properties, where <property> can be:
+Quarkus then allows the following JDBC-specific configuration of the object store included in `quarkus.transaction-manager.object-store._<property>_` properties, where _<property>_ can be:
 
-
-* `type` (_string_): Configure this property to `jdbc` to enable usage of a Quarkus JDBC datasource for transaction logging.
+* `type` (_string_): Configure this property to `jdbc` to enable usage of a Quarkus JDBC datasource for storing transaction logs.
 The default value is `file-system`.
+
 * `datasource` (_string_): Specify the name of the datasource for the transaction log storage.
 If no value is provided for the `datasource` property, Quarkus uses the xref:datasource.adoc#configure-datasources[default datasource].
+
 * `create-table` (_boolean_): When set to `true`, the transaction log table gets automatically created if it does not already exist.
 The default value is `false`.
+
 * `drop-table` (_boolean_): When set to `true`, the tables are dropped on startup if they already exist.
 The default value is `false`.
+
 * `table-prefix` (string): Specify the prefix for a related table name.
 The default value is `quarkus_`.
 
-// This paragraph will differ from the RHBQ docs in the future since it is not supported in the product yet.
-Additional information:
+For more configuration information, see the *Narayana JTA - Transaction manager* section of the Quarkus xref:all-config.adoc[All configuration options] reference.
 
-* You can manually create the transaction log table during the initial setup by setting the `create-table` property to `true`.
```

Comment:
Why did you remove "manually"? Could you reference it?

Reply:
@jmfinelli I asked for it. using [quarkus.transaction-manager.object-store.create-table](https://quarkus.io/guides/all-config#quarkus-narayana-jta_quarkus.transaction-manager.object-store.create-table) is a straight opposite to manually, it creates table **automatically**

Reply:
Makes sense to me.

Reply:
Thanks @MichalMaler.